src/plugins/loan-channel-query-map.js

```diff
@@ -170,37 +186,43 @@ export default {
 					id: 52,
 					url: 'loans-to-women',
 					queryParams: 'status=fundRaising&gender=female&riskRating=0,5',
-					algoliaParams: 'gender=female'
+					algoliaParams: 'gender=female',
+					flssLoanSearch: { gender: 'female' },
 				},
 				{
 					id: 53,
 					url: 'loans-for-education',
 					queryParams: 'status=fundRaising&riskRating=0,5&sector=15',
-					algoliaParams: 'sector=Education'
+					algoliaParams: 'sector=Education',
+					flssLoanSearch: { sector: [15] },
 				},
 				{
 					id: 54,
 					url: 'trending-now',
 					queryParams: 'status=fundRaising&riskRating=0,5',
-					algoliaParams: ''
+					algoliaParams: '',
+					flssLoanSearch: {},
```

Comment:
This will need to be updated to the `popularity` sort when it's available in FLSS.  It's not in the `queryParams` above b/c it's the default. We won't be able to include this category in the exp this go round.

Reply:
Removed.